tests/integration-tests/Apple/Helix.SDK.Tests.proj

```diff
@@ -9,15 +9,27 @@
     <HelixType>test/product/</HelixType>
     <IncludeXHarnessCli>true</IncludeXHarnessCli>
     <TestRunNamePrefix>$(AGENT_JOBNAME)</TestRunNamePrefix>
-    <EnableXUnitReporter>true</EnableXUnitReporter>
-    <EnableAzurePipelinesReporter>true</EnableAzurePipelinesReporter>
+    <EnableXUnitReporter Condition=" '$(AGENT_JOBNAME)' != '' ">true</EnableXUnitReporter>
```

Comment:
EnableXUnitReporter should just stay false, it doesn't actually get processed any more and can flake out and time people's tests out a tiny % of the time.

Reply:
Good to know! Removed from everywhere